runtime/frequency/src/lib.rs

```diff
@@ -1070,6 +1076,16 @@ impl_runtime_apis! {
 		fn metadata() -> OpaqueMetadata {
 			OpaqueMetadata::new(Runtime::metadata().into())
 		}
+
+		// REVIEW: substrate node template
+		fn metadata_at_version(version: u32) -> Option<OpaqueMetadata> {
```

Comment:
TODO: implementation to use these new apis.

Reply:
If the current metadata still works, I was thinking it should be a new issue.
Thoughts?

Reply:
I think this is just correct right? It matches the substrate node template and they all work:

<img width="682" alt="image" src="https://github.com/LibertyDSNP/frequency/assets/1252199/e994e2e0-89c6-4f8b-86ae-e92edb5300d1">